field_operations/update-all-fields.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Sets all non-title fields to optional.
+ */
+function hook_update_N() {
+  // Services we need.
+  $entityFieldManager = \Drupal::service('entity_field.manager');
+  $config = \Drupal::service('config.factory');
+
+  // Get all node fields.
+  $fields = $entityFieldManager->getFieldMap();
+  $fields = $fields['node'];
+
+  // Pop off the entity key & generic fields (nid, uuid, revision_log, etc).
+  $fields = array_slice($fields, 18, null, true);
+
+  // getFieldMap returns field names as the array key.
+  $keys = array_keys($fields);
+
+  // Loop through all fields, plus all bundles that a field is found on.
+  foreach ($keys as $key => $value) {
+    $field_name = $keys[$key];
+    $bundles = $fields[$value]['bundles'];
+
+    foreach ($bundles as $bundle) {
```

Comment:
Probably need a comment here to say something like "In this example, we will set each field instance to be optional"